pagopa/io-app
# fix: [IOBP-431] Idpay barcode double header

## Short description
fix of a navigation bug which showed double headers
<img src="https://github.com/pagopa/io-app/assets/60693085/7874d1c0-fe86-41b2-905f-0fba8baee677" width=200/>

## List of changes proposed in this pull request
- updated root navigator entry following recent changes

## How to test
with the dev-server active, navigate to a discount initiative and make sure no double header is rendered.
